Src/Support/GoogleApis.Auth.PlatformServices_Shared/OAuth2/LocalServerCodeReceiver.cs

```diff
@@ -375,10 +378,33 @@ private HttpListener StartListener()
             return listener;
         }
 
-        private async Task<AuthorizationCodeResponseUrl> GetResponseFromListener(HttpListener listener)
+        private async Task<AuthorizationCodeResponseUrl> GetResponseFromListener(HttpListener listener, CancellationToken ct)
         {
+            // Set up cancellation. HttpListener.GetContextAsync() doesn't accept a cancellation token,
```

Comment:
*\<sigh\>* I *looked* for something like that. How could I possibly have missed it :(